runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/ClueScrollOverlay.java

```diff
@@ -85,6 +90,16 @@ public Dimension render(Graphics2D graphics)
 			}
 		}
 
+		if (clue.isRequiresLight() && ((plugin.getInventoryItems() != null && !HAS_LIGHT.fulfilledBy(plugin.getInventoryItems()) || (plugin.getEquippedItems() != null && !HAS_LIGHT.fulfilledBy(plugin.getEquippedItems())))))
+		{
+			if (clue.getHasFirePit() != null && client.getVar(clue.getHasFirePit()) == 1)
```

Comment:
This can be combined with the conditions above? eg. `if (clue.isRequiresLight() && (... || (clue.getHasFirePit() != null && client.getVar(clue.getHasFirePit()) == 1)))`